lib/cloud_controller/deployment_updater/calculators/instances_to_scale_down.rb

```diff
@@ -0,0 +1,18 @@
+module VCAP::CloudController
+  module DeploymentUpdater
+    module Calculators
+      class InstancesToScaleDown
```

Comment:
**thought** I wonder if the calculators should just be dealing with numbers rather than deployments/processes, might make them easier to test, especially if the logic gets more knarly in the future e.g.

```Ruby
def initialize(max_in_flight, instances)
```